contributing.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to the playbook
+
+Contributions from the team are always welcome! This playbook is currently in alpha, meaning we're testing out what content and guidance is useful. Raising issues and pull requests is strongly encouraged, so that we can develop the playbook as a team.
+
+We continually update this playbook because we're always looking for better ways to get things done. And that changes over time, especially as who works on the team and what we work on changes too.
+
+This document is a guide on contributing to the team docs. It is a live guide and may not be complete, so you can raise problems or queries with a delivery manager or product manager.
+
+## Who can contribute
+
+Contributions must come from people on the GOV.UK Design System team. They are the users of these docs and the subject matter experts.
+
+## What to contribute
+
+This playbook is for you. It's a reference, a set of docs to turn to when you're unsure about anything, so contribute anything that will help you in your day job. 
```

Comment:
```suggestion
This playbook is for you. It's a reference, a set of docs to turn to when you're unsure about something, so contribute anything that will help you in your day job. 
```
A pedantic change, to note that we won't find answers on _anything_ but we will for some things. 😛

Reply:
Hahaha, not pedantic at all! Good spot.